installing/installing_rhv/installing-rhv-default.adoc

```diff
@@ -0,0 +1,71 @@
+[id="installing-rhv-default"]
```

Comment:
I kind of feel like we should mention to users that the default cluster will fail unless this is fixed: https://bugzilla.redhat.com/show_bug.cgi?id=1818577

Reply:
I expanded the following note in  installing-rhv-creating-custom-vm.adoc and added it below the `.Prerequisites` in:
- installing-rhv-customizations.adoc
- installing-rhv-default.adoc

`[IMPORTANT]
====
Creating a custom virtual machine template for {rh-virtualization} is a workaround for a known issue (link:https://bugzilla.redhat.com/show_bug.cgi?id=1818577[*BZ#1818577*]). If you do not create a custom virtual machine, the {product-title} cluster you install will fail.
====
`